CMakeLists.txt

```diff
@@ -49,6 +49,22 @@ if (${BUILD_VIEWER})
     COMMAND npm run build
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/view
   )
+  install(DIRECTORY view/build DESTINATION /var/lib/diffkemp/view
+    # Setting 755 for files.
+    FILE_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
+                     GROUP_READ GROUP_EXECUTE
+                     WORLD_READ WORLD_EXECUTE
+    # Setting 777 for directories to make it possible to create
+    # `src` and `diffs` folder by user (`diffkemp view`).
```

Comment:
These is quite a permissive access, maybe we could create empty `src/` and `diffs/` subdirs and only grant those 777 (and 755 to the rest)?

Related: do not install contents of `src/` and `diffs/`.

Reply:
711 for the rest (not `diffs/` and `src/`) looks like is enough.

Reply:
I'm not sure, you usually need R+E to do anything reasonable with a directory. I'd stick with the standard 755 for directories.

Reply:
Ok, to be safe I changed it to 755, but when I tested 711 did work too.